test/runtests.jl

```diff
@@ -10,6 +10,18 @@ else
     typemax(Csize_t)
 end
 
+if haskey(ENV, "JULIA_TEST_EXEFLAGS")
+    const test_exeflags = `$(split(ENV["JULIA_TEST_EXEFLAGS"]))`
+else
+    const test_exeflags = `--check-bounds=yes --startup-file=no --depwarn=error`
+end
+
+if haskey(ENV, "JULIA_TEST_EXENAME")
+    const test_exename = `$(split(ENV["JULIA_TEST_EXENAME"]))`
```

Comment:
should this be cleverer about shell-splitting if there's a `"path with spaces" --that-has-flags` ?